test/int32.test.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+const assert = require('assert');
+const start = require('./common');
+
+const mongoose = start.mongoose;
+const Schema = mongoose.Schema;
+
+describe('Int32', function() {
+  beforeEach(() => mongoose.deleteModel(/Test/));
+
+  it('is a valid schema type', function() {
+    const schema = new Schema({
+      myInt32: Schema.Types.Int32
+    });
+    const Test = mongoose.model('Test', schema);
+
+    const doc = new Test({
+      myInt32: 13
+    });
+    assert.strictEqual(doc.myInt32, 13);
+    assert.equal(typeof doc.myInt32, 'number');
+  });
+
+  it('supports the \'required\' property', async function() {
```

Comment:
(entirely optional)

Usually we'd just use a different outer quote:
```suggestion
  it(`supports the 'required' property`, async function() {
```